test/e2e/tests/dump_run.py

```diff
@@ -0,0 +1,120 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+
+import json
+import pathlib
+import sys
+
+
+def get_init_args():
+    return {
+        "kwargs": {
+            "project": "foo",
+        }
+    }
+
+
+def get_jobs():
+    litani_path = pathlib.Path(__file__).parent.parent.parent.parent / "litani"
```

Comment:
the `.parent.parent.parent.parent` is a bit of a red flag for me here. Is there a less hacky way of doing this?

Reply:
Yes :) I fixed this so that the top-level end-to-end test runner sets the path to Litani as an environment variable, since it already has that information.